app/uk/gov/hmrc/accessibilitystatementfrontend/views/AccessibilityStatementTemplate.scala.html

```diff
@@ -178,14 +178,14 @@ <h2 class="govuk-heading-l">@{messages("testing.heading", serviceOrApp)}</h2>
 } else {
     @if(statement.serviceLastTestedDate.isDefined) {
         @if(statement.automatedTestingOnly.getOrElse(false)) {
-            <p class="govuk-body">@{messages("testing.last-tested.automated", utils.Dates.formatDate(LocalDate.fromDateFields(statement.serviceLastTestedDate.get)), serviceOrApp)}</p>
+            <p class="govuk-body">@{messages("testing.last-tested.automated", utils.Dates.formatDate(dateToLocalDate(statement.serviceLastTestedDate.get)), serviceOrApp)}</p>
         } else {
-            <p class="govuk-body">@{messages("testing.last-tested", utils.Dates.formatDate(LocalDate.fromDateFields(statement.serviceLastTestedDate.get)), serviceOrApp)}</p>
+            <p class="govuk-body">@{messages("testing.last-tested", utils.Dates.formatDate(dateToLocalDate(statement.serviceLastTestedDate.get)), serviceOrApp)}</p>
         }
     }
     <p class="govuk-body">@{messages("testing.details.part1", serviceOrApp)}
         <a class="govuk-link" href="http://www.digitalaccessibilitycentre.org/">@{messages("testing.details.part2")}</a>@{messages(if(statement.automatedTestingOnly.getOrElse(false)) "general.fullstop" else "testing.details.part3", serviceOrApp)}
     </p>
 }
 
-<p class="govuk-body">@{messages("testing.created-and-updated", utils.Dates.formatDate(LocalDate.fromDateFields(statement.statementCreatedDate)), utils.Dates.formatDate(LocalDate.fromDateFields(statement.statementLastUpdatedDate)))}</p>
+<p class="govuk-body">@{messages("testing.created-and-updated", utils.Dates.formatDate(dateToLocalDate(statement.statementCreatedDate)), utils.Dates.formatDate(dateToLocalDate(statement.statementLastUpdatedDate)))}</p>
```

Comment:
Should we consider changing our model to use a local date rather than needing this helper? Is that possible?

Reply:
@matthewmascord I did consider that - I don't see any reason why we couldn't parse the YAML to read as `LocalDate` rather than `Date` - shall I make that change?

Reply:
Yeah, if you think it would help - should simplify it a bit?